.github/workflows/dip.yml

```
@@ -10,8 +10,8 @@ jobs:
         uses: actions/checkout@v2
       # yamllint disable rule:line-length
       - name: Download DIP
-        run: curl -L https://github.com/030/dip/releases/download/4.0.2/dip-ubuntu-20.04 -o dip && chmod +x dip
+        run: curl -L https://github.com/030/dip/releases/download/4.1.0/dip-ubuntu-20.04 -o dip && chmod +x dip
       - name: Check Alpine
-        run: ./dip image --name=alpine --regex=^3\.[0-9]+\.[0-9]+$ --dockerfile
+        run: ./dip image --name=alpine --regex=^3\.[0-9]+\.[0-9]+$ --updateDockerfile
```

This will update the Dockerfile, but it will not commit it or let the pipeline fail.

---

Example to commit the changes and to create a PR: https://github.com/030/yaam/blob/main/.github/workflows/dip.yml#L53